### DISCUSSION TITLE
pool corruption

### DISCUSSION THREAD

**Discussion by unkownerror1** [openzfs/zfs#14368] [Q&A]
There is no special operation on the storage pool, but data is written to the file system, and there may be a restart operation. I would like to ask if there is any way to locate this type of problem
![VIB1FUK4B0~ZPNIIIH3HZQB](https://user-images.githubusercontent.com/109565529/211455121-a377605f-1918-46b2-8bda-f81af0ca0561.png)

**Comment by Abassoo** [comment]
你看下这 [https://openzfs.github.io/openzfs-docs/msg/ZFS-8000-72/index.html](url)

**Reply by IvanVolosyuk** [reply]
The link URL is different from the text.

**Reply by Abassoo** [reply]
Clicking directly doesn't seem to work. You can copy links in new web pages to open.

**Comment by IvanVolosyuk** [comment]
https://openzfs.github.io/openzfs-docs/msg/ZFS-8000-72/index.html seem to work, the link from Abassoo doesn't.
